### PR TITLE
Set up enzyme testing

### PR DESCRIPTION
**Pull request by uxengine**

### CODE REVIEW

File: components/Me/index.test.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import test from 'tape'
+import { configure, shallow } from 'enzyme'
+import Adapter from 'enzyme-adapter-react-16'
+import { Me } from '../Me'
+
+configure({ adapter: new Adapter() })
```
Comment (tpreusse) [inline components/Me/index.test.js]: let's create a `lib/utils/enzyme.js` which does:

```
import Enzyme from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'

Enzyme.configure({ adapter: new Adapter() })

export * from 'enzyme'
export default Enzyme
```

Comment (tpreusse) [reply components/Me/index.test.js]: and then import it here

File: components/Me/index.test.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+test('utils.createActionButton', assert => {
```
Comment (tpreusse) [inline components/Me/index.test.js]: let's add a proper name

File: components/Me/index.test.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  assert.plan(1)
+
+  // TODO: Properly hook up withMe/withT here.
```
Comment (tpreusse) [inline components/Me/index.test.js]: IMO no need to test `withMe/withT` right now

Comment (tpreusse) [reply components/Me/index.test.js]: maybe a better test case would be popover, clicking close and testing that it is hidden

**Comment by tpreusse** [comment]
Reading the `next.js` 4.0.0 release notes I noticed a new features that allows to configure babel presets used by `next.js` and making them tape runner compatible:
https://github.com/zeit/next.js/pull/2991

Now we should have full next support in tests.

**Comment by uxengine** [comment]
Thanks for the improvements!

**Pull request merged**